index.html

```diff
@@ -48,45 +60,41 @@ <h1 class="pb-3"><p class="pt-0 fs-4 ">Hello! Students</p>
       <hr>
       <div class="d-flex flex-column flex-md-row">
-        <button class="btn d-flex flex-md-column ">
+        <div class="div-11 d-flex flex-md-column ">
           <i class="fa-solid fa-display fa-2xl text-white "></i>
           <h4 class="pt-2">Lecture</h4>
-          <div class="para">
-            <p class="text-white">Listen to the speakers from various countries about the messages of sharing and
-              opening.
-            </p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column ">
+          <p class="text-white ps-3 pb-2 para">Listen to the speakers from various countries about the messages of
+            sharing and
+            opening.
+          </p>
+        </div>
+        <div class="div-11 d-flex flex-md-column " >
           <i class="fa-solid fa-message fa-2xl text-white"></i>
           <h4 class="pt-2">Forum</h4>
-          <div class="para">
-            <p class="text-white ">Have the time to share your thoughts and opinions with expert for each topic.</p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column">
+          <p class="text-white para pb-2 ps-3">Have the time to share your thoughts and opinions with expert for each
+            topic.</p>
+        </div>
+        <div class="div-11 d-flex flex-md-column">
           <i class="fa-solid fa-desktop fa-2xl text-white"></i>
           <h4 class="pt-2">Workshop </h4>
-          <div class="para">
-            <p class="text-white">Trying creating your own work using open source license rather than just watching in
-              it.
-            </p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column ">
+          <p class="text-white ps-3 pb-2 para">Trying creating your own work using open source license rather than just
+            watching in
+            it.
+          </p>
+        </div>
+        <div class="div-11 d-flex flex-md-column ">
           <i class="fa-solid fa-code fa-2xl text-white"></i>
           <h4 class="pt-2">Challenge</h4>
-          <div class="para">
-            <p class="text-white">The questions in these coding challenges test your algorithms and data structure
-              knowledge.</p>
-          </div>
-        </button>
+          <p class="text-white ps-3 pb-2 para">The questions in these coding challenges test your algorithms and data
+            structure
+            knowledge.</p>
+        </div>
       </div>
-      <button class=" btn text-white p-3 mt-3"><u>SEE THE WHOLE PROGRAM</u></button>
+      <button class=" btn text-white p-3 pb-2 mt-3"><u>SEE THE WHOLE PROGRAM</u></button>
     </div>
   </section>
 
-  <section class="featureSpeaker pt-5" id="feature"></section>
+  <section class="featureSpeaker pt-5 " id="feature"></section>
   <section class="partner bg-dark pt-5">
     <h3 class="partner-h3 pb-3 text-center"> Partner</h3>
     <hr class="partner-hr pt-5">
```

Comment:
- [x] According to the original design, the `partners` and `footer` sections should not be shown in the **mobile version**, the last section for the `mobile version` should be the speakers. Please, kindly address this issue in the mobile version in order to match the design :smiley: :raised_hands:

| Original design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147131660-823d562f-6346-4262-9854-f7292853388d.png) | ![image](https://user-images.githubusercontent.com/45231276/181492853-7c14db06-1757-428f-9cee-bca17f3d1cce.png) |